src/pages/[userId]/prospects.js

```diff
@@ -1,54 +1,157 @@
+  }
+  .contact-col {
+    grid-column: 5/ span 2;
+  }
+  .notes-col {
+    grid-column: 7/ span 4;
+  }
+  .connections-col {
+    grid-column: 11/ span 2;
+  }
+
+  .dark {
+    background: #F5F5F5;
+
+  }
+
+  .numbers {
+    display: block;
+    width: 15px;
+    height: 15px;
+    text-align: center;
+    background-color: red;
+    color: white;
+    border-radius: 50%;
+    margin: 0 auto;
+  }
+
+@media (max-width: 800px){
+  font-size: .7rem;
+  text-align: center;
+
+  h1 {
+    margin-left: 0;
+  }
+
+  .name-col{
+    margin-left: -1rem;
+    grid-column: 1/ span 3;
+  }
+
+  .status-col {
+    grid-column: 4/ span 2;
+  }
+  .contact-col {
+    grid-column: 6/ span 4;
+  }
+  .connections-col {
+    grid-column: 10/ span 3;
+  }
+
+}
+  `;
+
   const prospects = [
     {
       id: 1,
-      first_name: 'Jon',
-      last_name: 'Smith',
-      notes: 'contacted on 5/17',
+      name: 'Jon Smith',
+      status: 'contacted',
+      last_contacted: '07/13',
+      notes: ['Meeting up on 08/27', 'Left a voicemail', 'Emailed'],
+      others: ['Reagann Smith', 'Ryan Bristow'],
+      pledge: false,
     },
     {
       id: 2,
-      first_name: 'Matt',
-      last_name: 'Clark',
-      notes: 'need to contact',
+      name: 'Matt Clark',
+      status: 'pledged, no amount',
+      last_contacted: '06/21',
+      notes: ['Need to follow up'],
+      others: ['Danielle Clark'],
+      pledge: true,
     },
     {
       id: 3,
-      first_name: 'Mitchell',
-      last_name: 'Pavel',
-      notes: 'No contact list',
+      name: 'Mitchell Pavel',
+      status: 'contacted',
+      last_contacted: '06/30',
+      notes: ['Wants to donate, partnering with the wife'],
+      others: [],
+      pledge: true,
     },
     {
       id: 4,
-      first_name: 'Bob',
-      last_name: 'Chin',
-      notes: 'contacted on 05/10',
+      name: 'Bob Chin',
+      status: 'No answer',
+      last_contacted: '07/01',
+      notes: ['Left voicemail', 'Attempted contact; No response.'],
+      others: [],
+      pledge: false,
     },
   ];
 
   return (
-    <>
-      <h1 style={{ marginLeft: '1rem' }}>Prospects for {userId}</h1>
-      <Grid cols={3}>
-        <h3>First Name:</h3>
-        <h3>Last Name:</h3>
-        <h3>Notes:</h3>
+    <Container>
+      <h1>Prospects for {userId}</h1>
+      <Grid className="columns" cols={12}>
+        <h4 className="name-col">Name</h4>
+        <h4 className="status-col">Status</h4>
+        <h4 className="contact-col">Last Contacted</h4>
+        <DesktopOnly>
+          <h4 className="notes-col">Notes</h4>
+        </DesktopOnly>
+
+        <h4 className="connections-col">Other Connections</h4>
       </Grid>
 
       {prospects.map((prospect) => (
-        <Grid key={prospect.id} cols={3}>
-          <h4>{prospect.first_name}</h4>
-          <h4>{prospect.last_name}</h4>
-          <h4>{prospect.notes}</h4>
+        <Grid className={prospect.id % 2 ? 'dark columns' : 'columns'} key={prospect.id} cols={12}>
```

Comment:
We don't want to use the prospect `id` for the zebra striping since it won't work later with real data.
Use this instead: https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child

Reply:
Very good point. I never use this feature of CSS and always forget it exists, but this is a perfect use case!

Done.